tasks/bio-read-quality/.mypy.ini

```diff
@@ -0,0 +1,27 @@
+# Global options:
+
+[mypy]
+python_version = 3.9
+pretty = True
+warn_return_any = True
+warn_no_return = True
+warn_redundant_casts = True
+warn_unused_configs = True
+warn_unused_ignores = True
+warn_unreachable = True
+disallow_subclassing_any = True
+disallow_untyped_calls = True
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+no_implicit_optional = True
+strict_optional = True
+allow_redefinition = False
+
+
+# Per-module options:
+# https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
+[mypy-matplotlib.*]
```

Comment:
matplotlib has package called `matplotlib-stubs` which gets installed by running `mypy --install-types` so it should not be present in this file